taccsite_cms/settings.py

```diff
@@ -255,6 +264,10 @@ def gettext(s): return s
     },
 ]
 
+LOCALE_PATHS = [
+    os.path.join(BASE_DIR, 'taccsite_cms', 'locale'),
+]
```

Comment:
**Note.** This is also added in #467 but with documentation about how to use `.po` files.